app/viewUtils/ReportingFrequencyViewUtils.scala

```diff
@@ -33,27 +35,60 @@ class ReportingFrequencyViewUtils @Inject()()(
 ) extends FeatureSwitching {
 
 
-  def itsaStatusString(itsaStatus: ITSAStatus)(implicit messages: Messages): Option[String] = {
+  def itsaStatusString(itsaStatus: ITSAStatus)(implicit messages: Messages, user: MtdItUser[_]): Option[String] = {
+    if(isEnabled(OptInOptOutContentUpdateR17)){
+      itsaStatus match {
+        case Mandated => Some(messages("reporting.frequency.table.mandated.r17"))
+        case Voluntary => Some(messages("reporting.frequency.table.voluntary.r17"))
+        case Annual => Some(messages("reporting.frequency.table.annual.r17"))
+        case _ => None
+      }
+    }else{
+      itsaStatus match {
+        case Mandated => Some(messages("reporting.frequency.table.mandated"))
+        case Voluntary => Some(messages("reporting.frequency.table.voluntary"))
+        case Annual => Some(messages("reporting.frequency.table.annual"))
+        case _ => None
+      }
+    }
+  }
+
+  private def isUsingMTD(itsaStatus: ITSAStatus)(implicit messages: Messages): Option[String] = {
     itsaStatus match {
-      case Mandated => Some(messages("reporting.frequency.table.mandated"))
-      case Voluntary => Some(messages("reporting.frequency.table.voluntary"))
-      case Annual => Some(messages("reporting.frequency.table.annual"))
+      case Mandated => Some(messages("reporting.frequency.table.MTD.voluntary"))
+      case Voluntary => Some(messages("reporting.frequency.table.MTD.voluntary"))
+      case Annual => Some(messages("reporting.frequency.table.MTD.annual"))
       case _ => None
     }
   }
 
-  def itsaStatusTable(optOutProposition: OptOutProposition)(implicit messages: Messages): Seq[(String, Option[String])] = {
+  def itsaStatusTable(optOutProposition: OptOutProposition)(implicit messages: Messages, user: MtdItUser[_]): Seq[(String, Option[String], Option[String])] = {
```

Comment:
we can use a case class here to replace the 3-tuple which we can then assign named values to represent each ._1, ._2, ._3

This would help with naming and clarity when it is being used.

Reply:
```case class ItsaStatusTableContent(taxYear: String, isUsingMTD:Option[String], status:isUsingMTD:Option[String])```

```def itsaStatusTable(optOutProposition: OptOutProposition)(implicit messages: Messages, user: MtdItUser[_]): Seq[ItsaStatusTableContent]```